schema/attribute.cpp

```diff
@@ -154,6 +154,18 @@ void Attribute::List::dump()
     }
 }
 
+bool operator==(const Attribute &lhs, const Attribute &rhs)
+{
+    return ( // XmlElement:
```

Comment:
Why not delegate that part to XmlElement::operator==?
(well, this is related to my next comment)